### ISSUE TITLE
Failure installing blurr, when dealing with the tranformers library

### ISSUE THREAD

**Issue by PabloAMC** [ohmeow/blurr#22]
I am having some trouble just installing both the transformers library from huggingface and blurr library. If I install first
```
!pip install transformers==3.5
!pip install fastai
```
The option to set `transformers == 3.5`, is because this seems to be related to the problem I find later, but it does not solver the problem. Then, after checking that everything is ok, for example using
```
import fastai
import transformers
print('fastai version :', fastai.__version__)
print('transformers version :', transformers.__version__)
```
I try to 
```
!pip install git+git://github.com/ohmeow/blurr.git@master
```
 or even simpler 
```
pip install ohmeow-blurr
```
The result from this is that I get
```
...
  Attempting uninstall: transformers
    Found existing installation: transformers 3.5.0
    Uninstalling transformers-3.5.0:
      Successfully uninstalled transformers-3.5.0
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
allennlp 1.2.2 requires transformers<3.6,>=3.4, but you have transformers 4.1.1 which is incompatible.
Successfully installed nlp-0.4.0 ohmeow-blurr-0.0.20 onnxruntime-1.6.0 rouge-score-0.0.4 seqeval-1.2.2 tokenizers-0.9.4 transformers-4.1.1 xxhash-2.0.0
```
I am running everything on a Kaggle notebook. Is there anything I am doing wrong, or is this even supposed to be a problem?
Thanks a lot!

**Comment by ohmeow** [comment]
Hey Pablo!

Looks like the issue may have something to do with allennlp (that's not a dependency in blurr) ... `allennlp 1.2.2 requires transformers<3.6,>=3.4, but you have transformers 4.1.1 which is incompatible`.

Have you tried updating it?

I'm working on a new release for blurr before end of year, and I'm mostly trying to keep things up-to-date with the latest transformers library, and so I'm not sure if the latest bits will work with transformers v.3.5.  You can try with your current setup and see ... lmk either way.

**Comment by PabloAMC** [comment]
Thanks a lot @ohmeow ! You're totally right that this is probably not an issue of blurr, but of the transformers and allen-nlp libraries. I posted it here because if I did not install blurr, but only fastai and transformers this error would not appear. I found [https://github.com/allenai/allennlp/commit/84a36a06e28894fe3fbf582186fd71a3eb25cd6f](this) but if I only installed fastai and blurr I would not get the error, even if that implicitly installs transformers (because I can query the version).
So in conclusion I am not sure why that was happening but it seems there is a way to avoid the error.
